rest/src/main/scala/io/udash/rest/RawRest.scala

```diff
@@ -0,0 +1,238 @@
+package io.udash
+package rest
+
+import java.util.concurrent.atomic.AtomicBoolean
+
+import com.avsystem.commons._
+import com.avsystem.commons.meta._
+import com.avsystem.commons.rpc._
+
+sealed abstract class RestMethodCall {
```

Comment:
There's a lot of internal stuff in `io.udash.rest` package. what about hiding it in `internat` package?

Reply:
Introduced `raw` package, but I'd like to leave these things public anyway.